runtime/src/main/kotlin/kotlin/native/Text.kt

```diff
@@ -9,52 +9,103 @@ package kotlin.native
 /**
  * Converts an UTF-8 array into a [String]. Replaces invalid input sequences with a default character.
  */
+@Deprecated("Use decodeToString or decodeZeroTerminatedToString instead")
```

Comment:
The message with two alternatives is fine, but I believe we should provide `ReplaceWith` to `decodeZeroTerminatedToString` as a behavior preserving option. Then one could erase "ZeroTerminated" if `decodeToString` is enough.

Reply:
Now two alternatives, `toKString` and `decodeToString`, are mentioned in message.
`ReplaceWith` holds `toKString`.